apps/protocol-frontend/src/components/VerificationFrameworkForm.tsx

```diff
@@ -63,15 +69,21 @@ const VerificationFrameworkForm = () => {
             isClearable
             localForm={localForm}
           />
-          <NumberInput
-            name="numberOfAttestors"
-            label="Choose the Number of Attestors"
-            defaultValue={1}
-            min={1}
-            max={10}
-            localForm={localForm}
-          />
-
+          {watch('verificationFramework') === 'none' ? (
```

Comment:
When I load the page, number of attestors is showing. It should by default be hidden unless number of attestors is selected.

Reply:
oh, interesting! i'll be sure to set the default to 'none' -- let me test that again. thanks!